apps/radix-ssr-testing/.eslintrc.json

```diff
@@ -28,6 +28,14 @@
                 "*.jsx"
             ],
             "rules": {}
+        },
+        {
+            "files": [
+                "*.ts"
+            ],
+            "rules": {
+                "@angular-eslint/prefer-standalone": "off"
```

Comment:
I am doubting we might want to switch it of - is this ok?

Reply:
I was confused by this at first too. It means that the linter will always remove the `standalone` flag since in Angular 19, `standalone: true` is the default.

In short, with this linter setting, we always create standalone elements! :)